src/common/options.cc

```diff
@@ -1310,6 +1310,23 @@ std::vector<Option> get_global_options() {
     .add_service("mon")
     .set_description("maximum number of OSDMaps to cache in memory"),
 
+    Option("mon_osd_cache_size_min", Option::TYPE_SIZE, Option::LEVEL_ADVANCED)
+    .set_default(128_M)
+    .add_service("mon")
+    .set_description("When cache auto-tuning is enabled, set the minimum amount of memory used for OSDMap caches."),
+
+    Option("mon_memory_target", Option::TYPE_SIZE, Option::LEVEL_BASIC)
+    .set_default(2_G)
+    .set_flag(Option::FLAG_RUNTIME)
+    .add_service("mon")
+    .set_description("The amount of bytes to be kept mapped in memory for OSDMaps with cache auto-tuning enabled"),
```

Comment:
the description should describe memory used for the monitor, rather than just osdmaps (it's already controlling rocksdb as well) - same for the mon_memory_autotune option

Reply:
I have modified the descriptions. Please verify.